js/index.js

```diff
@@ -182,25 +188,32 @@ window.addEventListener('load', () => {
       }
     });
   });
+  const registerList = JSON.parse(localStorage.getItem('register'));
+  contactMessage.value = registerList.message;
+  contactName.value = registerList.name;
+  email.value = registerList.email;
 });
 
 // My Recent Works
 
 // Form Valitation
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

Reply:
thank you for review